modules/virt-example-ethernet-nncp.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-ethernet-nncp_{context}"]
+= Example: Ethernet interface NodeNetworkConfigurationPolicy
+
+Create a ethernet interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest
+to the cluster.
+
+The following YAML file is an example of a manifest for an ethernet interface.
+It includes samples values that you must replace with your own information.
+
+[source,yaml]
+----
+
+apiVersion: nmstate.io/v1beta1
```

Comment:
question: I noticed that we're using `nmstate.io/v1beta1` whereas in previous example `nmstate.io/v1alpha1` is used. should we align them (may be in a separate PR/commit of course)

Reply:
Good catch. That update is scheduled for 2.6. Changing this to v1alpha1